packages/extension-chakra-storefront/src/components/with-commerce-sdk-react/with-commerce-sdk-react.tsx

```diff
@@ -34,20 +35,26 @@ type WithCommerceSDKReactProps = {
     logger: any
 }
 
+// Define a type for the server context
+type ServerContext = {
+    req: {
+        originalUrl?: string
+    }
+}
+
```

Comment:
How are we currently handling types in commerce-sdk-react.. does it export any? I'm wondering if that is the right place for that type.